content/en/docs/operations/talos/configuration/air-gapped.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Bootstrap a Talos Linux cluster for air-gapped environment
+linkTitle: air-gapped
+description: "Bootstrap a Talos Linux cluster for Cozystack with air-gapped"
+weight: 20
+aliases:
+  - /docs/talos/configuration/air-gapped
+---
+
+# Air-Gapped Installation of Talos Linux Cluster
+
+This guide outlines the steps to bootstrap a Cozystackin an **air-gapped environment**.
+
+**Air-gapped** means the cluster does **not** have direct access to the internet. All necessary resources (images, metadata, etc.) must be made available inside the local network.
+
+---
+## Configure nodes
+{{% alert color="warning" %}}
+For talm installation make this changes in `./templates/_helpers.tpl` before make template
+{{% /alert %}}
+
+### Configure NTP Servers
+
+Accurate time synchronization is critical for the cluster. In your Talos machine configuration, set **local NTP servers** that are accessible inside your private network.
+
+```yaml
+machine:
+  time:
+    servers:
+      - 192.168.0.4
+      - 10.10.0.5
+```
+Ensure the specified NTP servers are reachable from first node.
+
+### Configure Container Registry Mirrors
+Since the cluster cannot access public container registries, configure local registry mirrors for essential upstream registries.
```

Comment:
```suggestion
Since the cluster cannot access public container registries, it needs to use their local mirrors.
Creating such mirrors is out of the scope of this guide.
```